src/main/java/org/jabref/gui/collab/preamblechange/PreambleChangeDetailsView.java

```diff
@@ -0,0 +1,37 @@
+package org.jabref.gui.collab.preamblechange;
+
+import javafx.scene.control.Label;
+import javafx.scene.layout.VBox;
+
+import org.jabref.gui.collab.ExternalChangeDetailsView;
+import org.jabref.logic.bibtex.comparator.PreambleDiff;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.strings.StringUtil;
+
+public final class PreambleChangeDetailsView extends ExternalChangeDetailsView {
+
+    public PreambleChangeDetailsView(PreambleChange preambleChange) {
+        PreambleDiff preambleDiff = preambleChange.getPreambleDiff();
+
+        VBox container = new VBox();
+        Label header = new Label(Localization.lang("Changed preamble"));
+        header.getStyleClass().add("sectionHeader");
+        container.getChildren().add(header);
+
+        if (StringUtil.isNotBlank(preambleDiff.getOriginalPreamble())) {
+            container.getChildren().add(new Label(Localization.lang("Current value") + ": " + preambleDiff.getOriginalPreamble()));
```

Comment:
Since some languages may have different syntax i think its probably better to put the whole string into the localization including the colon and the value, e.g. `Localization.lang("Current value: %s ", preambleDiff.getOriginalPreamble())`